src/lib.rs

```diff
@@ -1,400 +1,171 @@
-#[cfg(feature = "tokio_block_in_place")]
-mod block_in_place;
+#![deny(missing_docs)]
+#![deny(missing_debug_implementations)]
+#![cfg_attr(docsrs, feature(doc_cfg))]
+#![cfg_attr(test, deny(warnings))]
+
+//! # vacation
+//!
+//!  `vacation``: give your (runtime) aworkers a break!
+//!
+//! ## Overview
+//!
+//! Today, when library authors are writing async APIs, they don't have a good way to handle long-running sync segments.
+//!
+//! For an application, they can use selective handling such as `tokio::task::spawn_blocking()` along with concurrency control to delegate sync segments to blocking threads. Or, they might send the work to a `rayon` threadpool.
```

Comment:
same typos here—maybe use the `#[include(readme.md)]` thing

Reply:
Oh, didn't know about that, thanks! Though I think that might have issues for adding the ignore flags?